database/sql/src/main/scala/cromwell/database/slick/tables/CallCachingHashComponent.scala

```diff
@@ -8,29 +8,27 @@ trait CallCachingHashComponent {
 
   import driver.api._
 
-  class CallCachingHashEntries(tag: Tag) extends Table[CallCachingHashEntry](tag, "CALL_CACHING_HASH") {
+  class CallCachingHashes(tag: Tag) extends Table[CallCachingHashEntry](tag, "CALL_CACHING_HASH") {
```

Comment:
ToL: I liked the informal rule of giving these database components an `Entries` name. Now there's this class `CallCachingHashes` and (IIRC) a totally separate `CallCacheHashes` which are at totally different layers of abstraction